integration-tests/oidc-wiremock/src/test/java/io/quarkus/it/keycloak/CodeFlowAuthorizationTest.java

```diff
@@ -515,6 +516,19 @@ public void testCodeFlowTokenIntrospection() throws Exception {
         clearCache();
     }
 
+    @Test
```

Comment:
Thanks for starting adding wiremock tests, each provider should also have a pair of unit tests, confirming all the properties have been set as expected,  and, that each property can be overridden, please see the known providers test in oidc/runtime

Reply:
Yes, I saw them. I can't say I can see value.  I will be confirming "is there what I set"? under what condition can such a test fail? I'll add it anyway, no point discussing this.